device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-C64/sai.profile

```diff
@@ -1,2 +1,4 @@
 SAI_INIT_CONFIG_FILE=/usr/share/sonic/hwsku/sai_3800.xml
 SAI_VXLAN_SRCPORT_RANGE_ENABLE=1
+SAI_DUMP_STORE_PATH=/var/log/mellanox/sdk-dumps
+SAI_DUMP_STORE_AMOUNT=10
```

Comment:
would you mind adding a new line here? maybe no functional impaction but let try to make all the files aligned.

Reply:
Fixed